pkg/diagnose/ports/ports.go

```diff
@@ -19,7 +20,7 @@ import (
 var agentNames = map[string]struct{}{
 	"datadog-agent": {}, "agent": {}, "trace-agent": {},
 	"process-agent": {}, "system-probe": {}, "security-agent": {},
-	"dogstatsd": {},
+	"dogstatsd": {}, "agent.exe": {}, "process-agent.exe": {}, "trace-agent.exe": {},
```

Comment:
i can take out `process-agent.exe` and `trace-agent.exe` but i think `agent.exe` still needs to be there because when i was testing it, the diagnose command won't recognize `agent.exe` as an agent process otherwise

Reply:
I meant update `isAgentProcess` to remove the `.exe` suffix
But if it's just adding a single entry in the list then no big deal 👍

Reply:
Ohh I see now. I modified `isAgentProcess` to strip `.exe` suffixes. Good idea